docker-compose.yml

```diff
@@ -1,12 +1,12 @@
 # Copyright 2018, 2019 Cyface GmbH
```

Comment:
The `eclipse_formatter.xml` file we use (latest version) automatically formats the 4 spaces to 2 spaces.

1) I think one reason is that in this file "Tab" was used with a default IDE setting of "equivalent to 4 spaces"
and my (default) IntelliJ settings use "tab equivalent to 2 spaces".

2) But what is weird:
In the BASE version of the xml file 4 spaces are used for the first 2 "tabs" and for the "deeper" ones only 1 space.

=> Can you please checkout the dev version of the collector (Commit b756dee42dc7f816923e9f6d8330950f60c543fe) and auto-format the docker-compose.yml file so that we see what happens with your IDE settings?